packages/paste-website/src/pages/form/input/index.mdx

```diff
@@ -0,0 +1,661 @@
+  {`<>
+  <FormLabel htmlFor="message_title" required>Message Title</FormLabel>
+  <FormInput
+    id="display_name"
+    name="display_name"
+    type="text"
+    placeholder="Ahoy, World"
+    insertAfter={
+      <Anchor href="#" display="flex">
+        <InformationIcon decorative={false} title="Get more info" />
+      </Anchor>
+    }
+    onChange=""
+  />
+  <FormHelperText>Please enter a short title for your message</FormHelperText>
+</>`}
+</LivePreview>
+
+### Input types
+
+How an input functions varies depending on the value of its type attribute. If a type attribute is not specified, the default type is text. The following type options are available:
+
+#### text
+
+- A single-line text input
+- Line-breaks are automatically removed from the input value
+- Applies type="text" by default
+
+#### email
+
+- An input for editing an email address
+- Looks like a text input, but has validation parameters and relevant keyboard in supporting browsers and devices with dynamic keyboards
+
+#### hidden
+
+- An input that is not displayed but whose value is submitted to the server
+
+#### number
+
+- An input for entering a number
+- Displays a stepper (small up/down arrows) and adds default validation when supported
+- Displays a numeric keypad in some devices with dynamic keypads
+
+#### password
+
+- An input whose value is obscured
+- Alerts user if the site is not secure
+
+#### readonly
+
+- An input that can't be edited but can receive focus
+- Allows users to highlight the text in the field.
+
+#### search
+
+- An input for entering search strings
+- Line-breaks are automatically removed from the input value
+- May include a delete icon in supporting browsers that can be used to clear the field
+- Displays a search icon instead of enter key on some devices with dynamic keypads
+
+#### tel
+
+- An input for entering a telephone number
+- Displays a telephone keypad in some devices with dynamic keypads
+
+## States
+
+### Input with inline error
+
+Change a form field to its error state and add an inline error if the value entered doesn't pass validation requirements.
+
+An inline error is placed below the field to inform a user of any errors in their value. If help text exists, the error text should replace and repeat the help text.
+
+<LivePreview scope={{FormInput, FormLabel, FormHelperText}} language="jsx">
+  {`<>
+  <FormLabel htmlFor="email_error">Email Address</FormLabel>
+  <FormInput id="email_error" name="email_error" type="email" placeholder="example@twilio.com" onChange="" hasError />
+  <FormHelperText variant="error">Please enter a valid email</FormHelperText>
+</>`}
+</LivePreview>
+
+### Disabled input
+
+Use a disabled form field to show users that they can't interact with the field.
+
+If you want to show information that can't be edited, use a read-only form field or consider another way of showing static information.
```

Comment:
Link to "read-only input": https://paste-670t0gn7a.now.sh/form/input/#read-only-input